### DISCUSSION TITLE
直角坐标系坐标轴数据都是0的情况下，如何设置轴线在最底部？

### DISCUSSION THREAD

**Discussion by Miss-An** [antvis/G2#6341] [Q&A]
### 问题描述

直角坐标系坐标轴数据最大值和最小值都是0的情况下，y轴的0刻度线是在中间，而不是在最下面

### 重现步骤

![Dingtalk_20240703094002](https://github.com/antvis/G2/assets/19668959/f784e393-05e7-4b2f-a1b7-74f2a05b1e0f)

### 预期行为

刻度线可以设置在最底部

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
`.scale('y', { domain: [0, 1] }`
